core/arch/opnd.h

```diff
@@ -842,6 +842,11 @@ enum {
 #    endif
 
     DR_NUM_GPR_REGS = DR_REG_STOP_GPR - DR_REG_START_GPR + 1,
+#    ifdef AARCH64
+    DR_NUM_SIMD_VECTOR_REGS = DR_REG_Z31 - DR_REG_Z0 + 1,
+#    else
+    DR_NUM_SIMD_VECTOR_REGS = 0,
```

Comment:
Wait, is this else path for 32-bit ARM?  It has SIMD registers.  This doesn't seem right.

Reply:
But only 64-bit right? Not what's commonly called "vector simd" like NEON. Can we add a FIXME comment here instead and leave it to the ARM guys?

Reply:
And re-open #3906 to address this?

Reply:
No, 128-bit.  We consider NEON part of the 32-bit ARM ISA in our decoder.  It has 128-bit SIMD registers.

Reply:
In this same file we define these SIMD registers: DR_REG_Q0...

Reply:
I think the proper number should just be put here now.  No reason to have someone else add a simple expression.

Reply:
Done, re-open if necessary/

Reply:
Ok, done.